generators/_init-python/index.js

```diff
@@ -101,7 +101,7 @@ class Generator extends Base {
     this.cwd = this.options.isWorkspace ? (config.cwd || config.name) + '/' : '';
     const deps = this._generateDependencies(useDevVersion.call(this, this.cwd), this.cwd);
 
-    patchPackageJSON.call(this, config, ['devDependencies'], null, null, this.cwd);
```

Comment:
Removed `['devDependencies']` in order to be able to add devDepenedecies to the python plugins. This is supposed to make sure that devDependencies are deleted for python plugins, but from my tests it is not necessary. The `package.json` looks the same before and after it. Added unit tests to test this.